package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "foreman_snapshot_management",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
I'd remove that one.

Reply:
@nadjaheitmann I have no strong opinion and knowledge on that. Can you discuss this with @maximiliankolb

Reply:
Just have a look at Katello version bumps. No bumping in package.json at all.